app/views/admin/shared/_sidenav_product.html.haml

```diff
@@ -4,6 +4,6 @@
   = tab "#{Item.model_name.human.pluralize} (#{current_facility.items.count})", facility_items_path(current_facility), sidenav_tab == Item.model_name.plural
   = tab "#{Bundle.model_name.human.pluralize} (#{current_facility.bundles.count})", facility_bundles_path(current_facility), sidenav_tab == Bundle.model_name.plural
 
-  - if current_ability.can?(:index, TrainingRequest)
+  - if SettingsHelper.feature_on?(:training_requests) && current_ability.can?(:index, TrainingRequest)
```

Comment:
I'd be tempted to move the settings check into `Ability`, but I could see that being a far less straightforward patch.